### PR TITLE
Jordan integration

### PR DESCRIPTION
**Pull request by pbxed**
submits GET request (w/hardcoded session-token) and retrieves a JSON showing all English premier league matches with win-draw-win markets, and odds.

### CODE REVIEW

**Comment by roioteromorales** [comment]
Take a look at Lombok. Is a framework that autogenerates getters, setters, constructors, hashcode, tostring...

File: src/main/java/com/pbx/matchbook_rest/data/repositories/MatchesRestRepository.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    private RestTemplate restTemplate;
+
+    @Autowired
+    public MatchesRestRepository(RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    @Override
+    public GameInformation getAllMatches() {
+        String url =    "https://api.matchbook.com/edge/rest/events?tag-url-names=premier-league&sport-ids=15" +
```
Comment (roioteromorales) [inline src/main/java/com/pbx/matchbook_rest/data/repositories/MatchesRestRepository.java]: https://stackoverflow.com/a/19540066

Comment (roioteromorales) [reply src/main/java/com/pbx/matchbook_rest/data/repositories/MatchesRestRepository.java]: A proper way to create a URL

File: src/main/java/com/pbx/matchbook_rest/presentation/MatchesRestController.java [hunk 2]
```diff
@@ -20,9 +18,11 @@ public MatchesRestController(MatchService matchService) {
     }
 
     @GetMapping("/matches")
-    public List<Match> getAllMatches() {
+    public GameInformation getGameInformation(Model model) {
+        GameInformation gameInformation = matchService.getAllMatches();
+        model.addAttribute("matches", gameInformation);
```
Comment (roioteromorales) [inline src/main/java/com/pbx/matchbook_rest/presentation/MatchesRestController.java]: what is this model for?

File: src/main/java/com/pbx/matchbook_rest/service/MatchService.java [hunk 3]
```diff
@@ -14,7 +13,7 @@ public MatchService(MatchesRepository matchesRepository) {
         this.matchesRepository = matchesRepository;
     }
 
-    public List<Match> getAllMatches() {
+    public GameInformation getAllMatches() {
```
Comment (roioteromorales) [inline src/main/java/com/pbx/matchbook_rest/service/MatchService.java]: if returns `GameInformation` then the method should be called `getGameInformation()`

File: src/main/java/com/pbx/matchbook_rest/presentation/MatchesRestController.java (hunk 2 above)
Comment (roioteromorales) [reply src/main/java/com/pbx/matchbook_rest/presentation/MatchesRestController.java]: This is a `RestController` you should have a normal `Controller` for the web

File: src/main/java/com/pbx/matchbook_rest/data/repositories/MatchesRestRepository.java (hunk 1 above)
Comment (pbxed) [reply src/main/java/com/pbx/matchbook_rest/data/repositories/MatchesRestRepository.java]: done

File: src/main/java/com/pbx/matchbook_rest/presentation/MatchesRestController.java (hunk 2 above)
Comment (pbxed) [reply src/main/java/com/pbx/matchbook_rest/presentation/MatchesRestController.java]: Changed to Controller. Updated model to retrieve list of events and display using thymeleaf

File: src/main/java/com/pbx/matchbook_rest/service/MatchService.java (hunk 3 above)
Comment (pbxed) [reply src/main/java/com/pbx/matchbook_rest/service/MatchService.java]: done

**Comment by pbxed** [comment]
Added lombok framework to project

**Pull request merged**